src/entities/applet/ui/AppletList.tsx

```diff
@@ -28,6 +29,14 @@ const AppletList: FC<BoxProps> = props => {
     );
   }
 
+  if (isSuccess && !applets?.length) {
+    return (
+      <XStack flex={1} jc="center" ai="center">
+        <NoListItemsYet text="applet_list_component:no_applets_yet" />
```

Comment:
I would argue that there should be a translated string passed in that prop. If we intend to pass a translation key we should name the prop accordingly to make it more obvious of what should be passed in.